netquery/module_api.go

```diff
@@ -0,0 +1,140 @@
+package netquery
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/safing/portbase/database"
+	"github.com/safing/portbase/database/query"
+	"github.com/safing/portbase/log"
+	"github.com/safing/portbase/modules"
+	"github.com/safing/portbase/runtime"
+	"github.com/safing/portmaster/network"
+)
+
+func init() {
+	var (
+		module   *modules.Module
+		db       *database.Interface
+		sqlStore *Database
+		mng      *Manager
+	)
+
+	module = modules.Register(
```

Comment:
This function call is _huge_. Consider moving prep and start to separate functions. Also, does anything need to be stopped or unloaded when shutting down (or prob done by workers)?